collect_app/src/main/java/org/odk/collect/android/widgets/UrlWidget.java

```diff
@@ -17,22 +17,23 @@
 import android.annotation.SuppressLint;
 import android.content.Context;
 import android.net.Uri;
+import android.util.TypedValue;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
 import android.widget.Button;
-import android.widget.LinearLayout;
 import android.widget.TextView;
 import android.widget.Toast;
 
 import org.javarosa.core.model.data.IAnswerData;
 import org.javarosa.core.model.data.StringData;
+import org.javarosa.form.api.FormEntryPrompt;
 import org.odk.collect.android.R;
 import org.odk.collect.android.formentry.questions.QuestionDetails;
-import org.odk.collect.android.formentry.questions.WidgetViewUtils;
 import org.odk.collect.android.utilities.CustomTabHelper;
+import org.odk.collect.android.utilities.MultiClickGuard;
 import org.odk.collect.android.widgets.interfaces.ButtonWidget;
 
-import static org.odk.collect.android.formentry.questions.WidgetViewUtils.createSimpleButton;
-import static org.odk.collect.android.formentry.questions.WidgetViewUtils.getCenteredAnswerTextView;
-
 /**
  * Widget that allows user to open URLs from within the form
```

Comment:
What do you think about such comments? Even on master (before this big change) there is no single line crated by Yaw (every single line has been edited) of course the whole project exists thanks to Yaw but isn't git enough?

Reply:
It was a question more to @lognaturel

Reply:
I think in this case it makes sense to remove the whole header comment since it doesn't add much. I also know for sure that @yanokwa doesn't like `@author` tags and is happy for history to be in `git`.

For those who use `@author` tags, the rule is supposed to be that anyone who touches the file adds their name and names are never removed. For that reason I don't typically remove author tags from people I don't know.

Reply:
Great! so @SaumiaSinghal please remove it.